store/postgres/src/chain_store.rs

```diff
@@ -1439,10 +1446,11 @@ impl ChainStoreTrait for ChainStore {
     ) -> Result<Vec<LightTransactionReceipt>, StoreError> {
         let pool = self.pool.clone();
         let storage = self.storage.clone();
-        let block_hash = block_hash.clone();
+        let blocks_table_path: String = storage.blocks_table();
```

Comment:
Since you are passing this to a function on `storage`, it doesn't need to be an argument on `storage.find_transaction_receipts_in_block` and that method can call it on `self`. That way, `blocks_table()` can be a private method on `Storage`